File: Python/tests/test_bart_withintrial_2par.py
import pytest

from hbayesdm.models import bart_withintrial_2par


def test_bart_withintrial_2par():
    _ = bart_withintrial_2par(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_withintrial_2par.R
context("Test bart_withintrial_2par")
library(hBayesDM)

test_that("Test bart_withintrial_2par", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_withintrial_2par(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
